main/create-window.js

```diff
@@ -86,9 +86,13 @@ module.exports = function createWindow (options) {
   }
 
   function updateTitle () {
-    var prefix = fromFile ? (path.basename(options.filePath) + ' - ') : ''
+    if (options.title) {
+      win.setTitle(options.title)
```

Comment:
@langpavel if it's not absolutely necessary to have it removed I'd prefer if the ` - vmd` suffix would stay. Apart from OS X most operating systems don't have a context based global menu. [Here's mine](http://i.imgur.com/jaLC5si.png). I don't have a menu or alt-tab-switch or anything so it'd be cool if the application name was in the title too.